cfgov/scripts/add_enforcement_data.py

```diff
@@ -0,0 +1,130 @@
+# Need court, institution type, docket number, tags(sp pop&prod), categories
+# status is record type
+
+import csv
+from datetime import datetime as dt
+from decimal import Decimal
+from re import sub
+
+from v1.models.learn_page import (
+    EnforcementActionDisposition, EnforcementActionPage
+)
+
+
+def strip_fields(f, *args):
+    return [f[i].strip() for i in args]
+
+
+def make_date(d):
+    return dt.strptime(d, '%m/%d/%y') if d else ''
+
+
+def money_to_dec(m):
+    s = sub(r'[^\d.]', '', m) or 0
+    return Decimal(s)
+
+
+record_to_status = {
+    'Final Order': 'post-order-post-judgment',
+    'Dismissal': 'expired-terminated-dismissed',
+    '': 'pending-litigation'
+}
+
+cdata = {}
+rdata = {}
+
+
+def add_relief(data_file='./cfgov/scripts/pea.csv'):
```

Comment:
I kinda don't like parameterizing scrips that are explicitly one-offs, but since we will probably be reusing&modifying this onec we get the final data anyway, added in 4bfeef4724dd83ff7d66003aa42e910d82dea80b (and your point about sudo is a good one)